rotkehlchen/tests/unit/decoders/test_curve_lend.py

```diff
@@ -0,0 +1,694 @@
+        decimals=18,
+        symbol='crvUSD',
+    )
+    vault_token = get_or_create_evm_token(
+        userdb=database,
+        evm_address=string_to_evm_address('0xccd37EB6374Ae5b1f0b85ac97eFf14770e0D0063'),
+        chain_id=ChainID.ETHEREUM,
+        protocol=CURVE_VAULTS_PROTOCOL,
+        name='Borrow crvUSD (WBTC collateral)',
+        symbol='cvcrvUSD',
+        underlying_tokens=[UnderlyingToken(
+            address=underlying_token.evm_address,
+            token_kind=EvmTokenKind.ERC20,
+            weight=ONE,
+        )],
+    )
+    return vault_token, underlying_token
+
+
+def _create_optimism_vault_tokens(database: 'DBHandler') -> tuple['EvmToken', 'EvmToken']:
+    """Ensure that vault tokens needed for the optimism tests are in the database.
+    Returns the vault token and underlying token in a tuple.
+    """
+    underlying_token = get_or_create_evm_token(
+        userdb=database,
+        evm_address=string_to_evm_address('0xC52D7F23a2e460248Db6eE192Cb23dD12bDDCbf6'),
+        chain_id=ChainID.OPTIMISM,
+        decimals=18,
+        symbol='crvUSD',
+    )
+    vault_token = get_or_create_evm_token(
+        userdb=database,
+        evm_address=string_to_evm_address('0xE8dd743E376EcA40cc9547236D46eD83e06Fd471'),
+        chain_id=ChainID.OPTIMISM,
+        protocol=CURVE_VAULTS_PROTOCOL,
+        name='Borrow crvUSD (WETH collateral)',
+        symbol='cvcrvUSD',
+        underlying_tokens=[UnderlyingToken(
+            address=underlying_token.evm_address,
+            token_kind=EvmTokenKind.ERC20,
+            weight=ONE,
+        )],
+    )
+    return vault_token, underlying_token
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('arbitrum_one_accounts', [['0x706A70067BE19BdadBea3600Db0626859Ff25D74']])
+def test_vault_deposit(
+        arbitrum_one_inquirer: 'ArbitrumOneInquirer',
+        arbitrum_one_accounts: list['ChecksumEvmAddress'],
+) -> None:
+    tx_hash = deserialize_evm_tx_hash('0xd7e237db5c10970fd4a5a8bccfc87412fad75680d1358a5d39b4a76ba9ffafa6')  # noqa: E501
+    vault_token, underlying_token = _create_arbitrum_vault_tokens(database=arbitrum_one_inquirer.database)  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=arbitrum_one_inquirer, tx_hash=tx_hash)  # noqa: E501
+    timestamp, user_address, gas_amount, approve_amount, deposit_amount, receive_amount = TimestampMS(1732052219000), arbitrum_one_accounts[0], '0.00000265801', '115792089237316195423570985008687907853269984665640564039400.04103445561341153', '57.542973457516228405', '55862.682306216187960829'  # noqa: E501
+    assert events == [
+        EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_amount)),
```

Comment:
I have said it multiple time. Please for single arg constructors keep it simple. Here and in all your tests in this file or in the future
```suggestion
            balance=Balance(FVal(gas_amount)),
```